esp/src/src-react/hooks/grid.tsx

```diff
@@ -120,7 +120,7 @@ function columnsAdapter(columns: DojoColumns, sorted?: QuerySortItem): IColumn[]
                 key,
                 name: column.label ?? key,
                 fieldName: column.field ?? key,
-                minWidth: column.width,
+                minWidth: column.width ?? 70,
                 maxWidth: column.width,
```

Comment:
Does maxWidth prevent column sizing?

Reply:
Not that I have seen. Though if it's removed, the initial width of any narrow (icon-labeled) columns is too large.